Cesium.Compiler/Compilation.cs

```diff
@@ -83,7 +82,8 @@ private static async Task GenerateCode(AssemblyContext context, string inputFile
         if (lastUnprocessedToken.Kind != CTokenType.End)
             throw new ParseException($"Excessive output after the end of a translation unit {inputFilePath} at {lexer.Position}.");
 
-        context.EmitTranslationUnit(translationUnit);
+        var translationUnitName = Path.GetFileNameWithoutExtension(inputFilePath);
+        context.EmitTranslationUnit(translationUnitName, translationUnit);
```

Comment:
What if we have two filenames in different folders?

Reply:
Good catch. We'll have to invent something to turn them apart in such case.

Reply:
Covered by #345.